projects/klippa/ngx-enhancy-forms/src/lib/form/form-validation-error/form-validation-error.ts

```diff
@@ -0,0 +1,36 @@
+import { InjectionToken } from '@angular/core';
+
+export class FormValidationError extends Error {
+	_path: string;
+	_message: string;
```

Comment:
- missing public/private.
- What is the reason for starting with `_`?

Reply:
but I've removed them in my new changes.